src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -518,6 +537,29 @@ void Increment(ref long counter)
         GC.KeepAlive(listener);
     }
 
+    /// <summary>Client context containing information pertaining to a single worker.</summary>
+    private sealed class ClientContext
+    {
+        public int TaskNum { get; }
+        public HttpClient HttpClient { get; }
+        public Random Random { get; }
+
+        public ClientContext(HttpClient httpClient, int taskNum, int seed)
+        {
+            TaskNum = taskNum;
+            HttpClient = httpClient;
+            // Random instance deriving from global seed and worker number
+            Random = new Random(Combine(seed, taskNum));
+
+            // deterministic hashing copied from System.Runtime.Hashing
+            int Combine(int h1, int h2)
+            {
+                uint rol5 = ((uint)h1 << 5) | ((uint)h1 >> 27);
+                return ((int)rol5 + h1) ^ h2;
+            }
```

Comment:
You can just use HashCode.Combine.

Reply:
`HashCode.Combine` mixes in random data. Wanted cross-process deterministic seed generation.

Reply:
That's fair.  At the same time, though, this isn't the only source of randomness in the test.  Its very nature of executing many things concurrently, and talking to a server, means there's already a lot of randomness in the execution, and it's extremely unlikely you'd get the same pattern of execution on multiple runs even with the same seed used.

Reply:
I know. I just figured eliminating randomness from the request sequence might make things simpler when evaluating issues.